Link to online compiler playground #44

henilp105 opened the pull request:
The Fortran-lang playground (https://github.com/fortran-lang/playground) should be linked from the webpage to make it easier to discover. Please refer #44   for more details. 

Possible options are:

- in the top navigation bar
- in the compiler section next to the supported compilers

CC @awvwgk @ashirrwad, @everythingfunctional @milancurcic

henilp105 commented:
#build_preview

henilp105 commented:
I have currently added it in the top navigation bar , But happy to change according to the suggestions.

henilp105 commented:
#build_preview

henilp105 commented:
#delete_preview

awvwgk commented on source/index.md:
```diff
@@ -150,6 +150,7 @@ Or just write Fortran software for your research, business, or schoolwork. You c
 :::{toctree}
 :hidden:
 
+Fortran Playground <https://play.fortran-lang.org/>
```
Repeating Fortran here should not be necessary

```suggestion
Playground <https://play.fortran-lang.org/>
```

henilp105 replied:
👍

henilp105 commented:
#build_preview

milancurcic commented on source/index.md:
```diff
@@ -150,6 +150,7 @@ Or just write Fortran software for your research, business, or schoolwork. You c
 :::{toctree}
 :hidden:
 
+Playground <https://play.fortran-lang.org/>
```
```suggestion
Play <https://play.fortran-lang.org/>
```

Rationale:

* The navbar is already busy and "Playground" is a mouthful
* "Play" is a verb so it has a more active voice;
* It flows better next to "Learn"

awvwgk replied:
I was looking at the Rust page where they use *Playground*, but I like *Play* better. I'm not sure whether we can insert an actual triangle here (maybe with UTF8?) to make it stand out more.

milancurcic reviewed (approved):
I left a suggestion to reword the link.

It'd also be nice to add a short paragraph and a button link in the Getting Started section of the Learn page, following the "Quickstart tutorial" and "Fortran-lang Discourse" links. What do you think about something like:

---
▶ Play with Fortran

Get a taste of Fortran in an interactive playground in the browser.

Button: [Fortran Playground]
---

henilp105 commented:
@awvwgk I have added Play with fortran section in learn, I have removed request by mistake, Would like to request review from all @milancurcic @ashirrwad @everythingfunctional .

Thanks and Regards,
Henil

henilp105 commented:
#build_preview

milancurcic reviewed (approved):
Thank you, I think it's good to go.

everythingfunctional reviewed (approved):
LGTM

milancurcic commented:
Thank you @henilp105 and reviewers!

milancurcic commented:
#delete_preview
